README.md

```diff
@@ -56,7 +56,7 @@ Valid vulnerabilities on any product not explicitly listed in scope will be acce
     * Secure coding best practice not used
     * TLS configuration with older ciphersuites
     * Host enumeration (e.g. via Semi-blind SSRF)
-    * Logout CSRF
+    * CSRF
```

Comment:
What about excluding "CSRF with only an Availability impact" ?

Reply:
That's reasonable, see https://github.com/grafana/bugbounty/pull/9/commits/ee906f2b68d26f4673613ccc70ff64e8114efc27 for that change.